src/ott/geometry/regularizers.py

```diff
@@ -305,55 +335,41 @@ def tree_flatten(self):  # noqa: D102
 
 @jtu.register_pytree_node_class
 class L1(ProximalOperator):
-  r"""L1 norm regularizer :math:`\lambda \|x\|_1`.
-
-  Args:
-    lam: Strength of the regularizer.
-  """
-
-  def __init__(self, lam: float = 1.0):
-    super().__init__()
-    self.lam = lam
+  r"""L1 norm regularizer :math:`\|x\|_1`."""
```

Comment:
i know this was this way before, but if we're using math-scripting, maybe `:math:\ell_1` instead of L1?

Reply:
Done.